cli/src/pcluster/validators/cluster_validators.py

```diff
@@ -561,6 +563,27 @@ def _validate(self, efs_id, head_node_avail_zone: str):
                 )
 
 
+class SharedStorageNameValidator(Validator):
+    """
+    Shared storage name validator.
+
+    Validate if the provided name for the shared storage complies with the acceptable pattern.
+    Since the storage name is used as a tag, the provided name must comply with the tag pattern.
+    """
+
+    def _validate(self, name: str):
+        if not re.match(PCLUSTER_SHARED_STORAGE_NAME_REGEX % PCLUSTER_SHARED_STORAGE_NAME_MAX_LENGTH, name):
```

Comment:
The regex is only used here. Can you avoid defining a constant for it? It makes it difficult to know what the resulting regex is here.

Reply:
The regex is the standard regex for a tag value. 
I think it makes sense to have it defined as a constant, as it has been done for other regex because it can be useful somewhere else.
I will rename the variable with a name that gives the idea that it is meant for tagging values.